espflash/src/cli/mod.rs

```diff
@@ -307,6 +312,23 @@ pub fn flash_elf_image(
         None
     };
 
+    if erase_otadata {
+        let partition_table = match &partition_table {
+            Some(partition_table) => partition_table,
+            None => return Err((MissingPartitionTable {}).into()),
+        };
+
+        let otadata = match partition_table.find("otadata") {
```

Comment:
I don't believe it's _required_ for this partition to be named `otadata`, only that it has type `data` and subtype `ota`. The first line of the [OTA Data Partition](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/ota.html#ota-data-partition) documentation seems to indicate this is the case.